### DISCUSSION TITLE
p2p_S1_TOPS_Frame.csh  error in unwrap section

### DISCUSSION THREAD

**Discussion by HPUdabiao** [gmtsar/gmtsar#769] [Q&A]
When I use P2P_ S1_ TOPS_ Frame.csh processed Sentinel 1A radar data and show an error in the unwrapping section. The following is the processing process. I am a beginner in gmtSAR and am seeking help from a teacher.
I modified the configuration file config.s1a.txt and changed the coherence threshold to 0.3, because if use the default parameter of 0, the unwrapping part will not work.other parameters is default

![image](https://github.com/gmtsar/gmtsar/assets/140577628/efc89d0f-4947-43e9-a494-8819c3b3cb8d)
![image](https://github.com/gmtsar/gmtsar/assets/140577628/039c94d5-74db-4745-9f3f-81b11e0b8e0c)
![image](https://github.com/gmtsar/gmtsar/assets/140577628/d6b1df44-f351-4b5d-8db6-2df0cea6f633)

**Comment by Xiaohua-Eric-Xu** [comment]
It says `Killed`. Could it be due to your machine not having enough RAM?

**Reply by HPUdabiao** [reply]
yes,you are right.Teacher.I solve this problem by using another computer with bigger RAM(16G).Thanks for your answer!

**Reply by Xiaohua-Eric-Xu** [reply]
You could potentially run this in sequential mode (0) to avoid lots of usage of memory.
